src/test/java/uk/gov/hmcts/divorce/ciccase/model/access/OrganisationPolicyAccessTest.java

```diff
@@ -28,12 +27,11 @@ void shouldGrantOrganisationPolicyAccess() {
         final SetMultimap<HasRole, Permission> grants = new OrganisationPolicyAccess().getGrants();
 
```

Comment:
Could this be a static method? What does instantiating a new OrganisationPolicyAccess object each time achieve?

Reply:
it is outside of context of current task but we can definitely speak on that in sperate session as we have so many similar implementations